taxcalc/consumption.json

```diff
@@ -0,0 +1,41 @@
+{
+    "_MPC_e17500": {
+        "start_year": 2013,
+        "long_name": "Marginal propensity to consume medical expenses",
+        "description": "Defined as dollar change in medical-expense consumption divided by dollar change in income.  Must be in [0,1] range.",
```

Comment:
Dan @feenberg said:
> I agree, but there will be people entering percents instead of fractions who would be helped by this restriction.

That is exactly the reason for the restriction: many are likely to enter 5 percent rather than 0.05.

@MattHJensen